Dockerfile.in

```diff
@@ -8,3 +8,10 @@ RUN helm init -c && \
     helm plugin install https://github.com/hypnoglow/helm-s3.git && \
     rm -f /var/cache/apk/*
 RUN helm repo add infobloxcto s3://infoblox-helm-dev/charts
```

Comment:
What's the point of adding this repo is the image is discarded? I think there are jobs that depend on this repo already existing in the image.